homeassistant/components/starline/config_flow.py

```diff
@@ -0,0 +1,274 @@
+"""Config flow to configure StarLine component."""
+from typing import Optional
+from starline import StarlineAuth
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD
+from homeassistant.core import callback
+
+from .const import (
+    DOMAIN,
+    CONF_APP_ID,
+    CONF_APP_SECRET,
+    CONF_MFA_CODE,
+    CONF_CAPTCHA_CODE,
+    LOGGER,
+    CONF_SCAN_INTERVAL,
+    DEFAULT_SCAN_INTERVAL,
+    ERROR_AUTH_APP,
+    ERROR_AUTH_USER,
+    ERROR_AUTH_MFA,
+    DATA_USER_ID,
+    DATA_SLNET_TOKEN,
+    DATA_SLID_TOKEN,
+    DATA_EXPIRES,
+)
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class StarlineFlowHandler(config_entries.ConfigFlow):
```

Comment:
```suggestion
class StarlineFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
```

Reply:
With this change i have pylint error
`homeassistant/components/starline/config_flow.py:10:0: W0611: Unused DOMAIN imported from const (unused-import)`
How to fix it?

Reply:
Yeah, it's a bug in pylint. We can disable that warning locally on that line.